CHANGELOG.rst

```diff
@@ -1,6 +1,28 @@
 Changelog
 =========
 
+
+1.15.1 (2023-03-09)
+-------------------
+
+Bug Fixes
+"""""""""
+
+- Support retriveing extra fields which do not have extra prefix set in ``AzureDataFactoryHookAsync``.
+  (`#899 <https://github.com/astronomer/astronomer-providers/pull/899>`_)
+- Pass connection_id in ``HttpTrigger``.
+  (`#896 <https://github.com/astronomer/astronomer-providers/pull/896>`_)
+- Fix ``SFTPSensorAsync`` using newer_than doesn't work well with file_pattern and
```

Comment:
needs to be a user facing description

Reply:
can you elaborate on what you mean by not working well with file_pattern

Reply:
added details.